addons/hr_expense/wizard/account_register_payments_views.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_account_payment_from_invoices_inherit_hr_expense" model="ir.ui.view">
+        <field name="name">account.register.payments.wizard.inherited</field>
+        <field name="model">account.register.payments</field>
+        <field name="inherit_id" ref="account.view_account_payment_from_invoices"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <field name="partner_id" position="attributes">
```

Comment:
@RomainLibert I dont think it will make it technically different.
because we always have an inherit_id field in view xml if the view is inherited.
so it doesn't make any difference and if it is required then i can make the changes

Reply:
It is not a technical change, more of a readability change (which is why I don't block the task in the pipe)

Reply:
Then we wiil go for readability. :+1: